src/main/groovy/org/mockito/release/internal/gradle/ContinuousDeliveryPlugin.java

```diff
@@ -46,6 +46,19 @@ public void apply(final Project project) {
         project.getPlugins().apply(VersioningPlugin.class);
         project.getPlugins().apply(GitPlugin.class);
 
+        project.allprojects(new Action<Project>() {
+            @Override
+            public void execute(final Project project) {
+                project.getPlugins().withType(JavaLibraryPlugin.class, new Action<JavaLibraryPlugin>() {
+                    @Override
+                    public void execute(JavaLibraryPlugin plugin) {
+                        project.getTasks().getByName("publishToMavenLocal")
+                                .dependsOn(":fetchAllProjectContributorsFromGitHub");
```

Comment:
This is making an assumption that the root project will have "fetchAllProjectContributorsFromGitHub" task. I suggest to make it a little bit more flexible:
1. ContinuousDeliveryPlugin can apply the ContributorsPlugin that adds fetchAllProjectContributorsFromGitHub task
2. Then, here you can just configure

//subproject - project in the 
subproject.getTasks().getByName("publishToMavenLocal").dependsOn(project.getTasks().getByName("fetcher")

It's a lot of writing... let me just push a commit on your branch :)